docs/tutorials/11_VarQTE.ipynb

```diff
@@ -0,0 +1,957 @@
+    "\n",
+    "and\n",
+    "\n",
+    "$$V_{i}=\\begin{cases}\n",
+    "\\text{Im}\\left[\\frac{\\partial\\left\\langle \\psi\\left[\\theta\\right]\\right|}{\\partial\\theta_{i}}H\\left|\\psi\\left[\\theta\\right]\\right\\rangle +\\left\\langle \\psi\\left[\\theta\\right]\\right|\\frac{\\partial\\left|\\psi\\left[\\theta\\right]\\right\\rangle }{\\partial\\theta_{i}}\\left\\langle H\\right\\rangle _{\\theta}\\right] & \\text{Real time}\\\\\n",
+    "-\\text{Re}\\left[\\frac{\\partial\\left\\langle \\psi\\left[\\theta\\right]\\right|}{\\partial\\theta_{i}}H\\left|\\psi\\left[\\theta\\right]\\right\\rangle \\right] & \\text{Imaginary time}\n",
+    "\\end{cases}$$."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "caoBgyluhQ3L"
+   },
+   "source": [
+    "## Running VarQTE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "WG-18slyLnVM"
+   },
+   "source": [
+    "In this tutorial, we will use two classes that extend `VarQTE`, `VarQITE` ([Variational Quantum Imaginary Time Evolution](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.VarQITE.html#qiskit_algorithms.VarQITE)) and `VarQRTE` ([Variational Quantum Real Time Evolution](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.VarQRTE.html)) for time evolution.\n",
+    "We can use a simple Ising model on a spin chain to illustrate this. Let us consider the following Hamiltonian:\n",
+    "\n",
+    "$$H = -J\\sum_{i=0}^{L-2}Z_{i}Z_{i+1} - h\\sum_{i=0}^{L-1}X_i$$\n",
+    "\n",
+    "where $J$ stands for the interaction energy, and\n",
+    "$h$ represents an external field which is orthogonal to the transverse direction. $Z_i$ and $X_i$ are the Pauli operators on the spins. Taking $L=2$, $J=0.2$,  and $h =1$, the Hamiltonian and the magnetization $\\sum_i Z_i$ can be constructed using `SparsePauliOp` as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "id": "6B0nZEDh0EqI"
+   },
+   "outputs": [],
+   "source": [
+    "from qiskit.quantum_info import SparsePauliOp\n",
+    "\n",
+    "hamiltonian = SparsePauliOp(['ZZ',  'IX', 'XI'],\n",
+    "              coeffs=[-0.2 , -1, -1])\n",
+    "\n",
+    "magnetization = SparsePauliOp([ 'IZ', 'ZI'], coeffs=[1, 1])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Y_9Jskw4m8nF"
+   },
+   "source": [
+    "# Imaginary Time Evolution\n",
+    "\n",
+    "Imaginary time evolution can be used, for example, to find the ground state or calculate the finite temperature expectation value of the system. Here, we will use the `VarQITE` class from `algorithms.time_evolvers.variational` to compute a ground state energy. Firstly, we need to choose an ansatz. We can use `EfficientSU2` to easily construct an ansatz, setting the number of repetitions using `reps`."
```

Comment:
```suggestion
    "Imaginary time evolution can be used, for example, to find the ground state or calculate the finite temperature expectation value of the system. Here, we will use the `VarQITE` class from `qiskit_algorithms.time_evolvers.variational` to compute a ground state energy. Firstly, we need to choose an ansatz. We can use `EfficientSU2` to easily construct an ansatz, setting the number of repetitions using `reps`."
```
Given we provided links, just above, to the classes, which included this we could probably drop the where its from aspect here